action.yaml

```diff
@@ -159,12 +159,13 @@ runs:
           local var_name="$1"
           local input_value="$2"
           local default_value="$3"
-        
-          if [ ! -z "$input_value" ] && [ "$input_value" != "$default_value" ]; then
+          local value="${!var_name}"
+
+          if [ ! -z "$input_value" ] && [ "$input_value" != "$default_value" ] || [ "$value" != "$default_value" ]; then
```

Comment:
This helps in your case, but breaks logic for config file.
Let's look at an example - if we don't set `format` in `inputs` - we should use format from config file (`json` in this example):
- master branch - `json` format - https://github.com/DmitriyLewen/test-trivy-action/actions/runs/12134369796/job/33831459219#step:3:130
- your changes - `table` format - https://github.com/DmitriyLewen/test-trivy-action/actions/runs/12134373819/job/33831470667#step:3:146

Reply:
What if we will `unset` all `TRIVY_*` envs after run Trivy?